src/App.tsx

```diff
@@ -1,19 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import Header from './components/Header/Header';
 import Cards from './components/Card/Cards';
 import initialData from './state/CardData';
+import Modal from './components/Card/CardModal/Modal';
 
-const App = () => (
-  // const [initialData, setNewData] = useState('CARD_DATA');
-  <div className="app-wrapper">
-    <Header />
-    <div className="app-content">
-      {initialData.map((data) => (
-        <Cards key={data.id} title={data.title} text={data.text} />
-      ))}
+const App = () => {
+  const [modalActive, setModalActive] = useState(false);
+  const [cardList, setCardList] = useState(initialData);
+  const addCardHandler = (enteredTitle: any, enteredContent: any) => {
+    setCardList((prevCardList) => [
+      ...prevCardList,
+      { id: Math.random().toString(), title: enteredTitle, text: enteredContent },
```

Comment:
Please don't use Math.random - it's quite a bad practice, because it's not guaranteed unique. Instead of it please use a uuid library or any another similar to this (to create really unique ids).